compiler/rustc_expand/src/proc_macro_server.rs

```diff
@@ -406,7 +407,7 @@ impl server::FreeFunctions for Rustc<'_> {
     }
 
     fn track_path(&mut self, path: &str) {
-        self.sess.file_depinfo.borrow_mut().insert(Symbol::intern(path));
+        self.sess.file_depinfo.borrow_mut().insert(path::PathBuf::from(path));
```

Comment:
```suggestion
        self.sess.file_depinfo.borrow_mut().insert(PathBuf::from(path));
```
Nit (here and in the other files as well): types are usually imported directly, without parent module.